javalin-plugins/javalin-swagger-plugin/src/main/kotlin/io/javalin/openapi/plugin/swagger/SwaggerHandler.kt

```diff
@@ -35,6 +37,9 @@ class SwaggerHandler(
             .loadVersions()
             .joinToString(separator = ",\n") { "{ name: '$it', url: '$publicDocumentationPath?v=$it' }" }
 
+        val allCustomStylesheets = customStylesheetFiles.joinToString(separator = "\n")
+        val allCustomJavaScripts = customJavaScriptFiles.joinToString(separator = "\n")
```

Comment:
I'd personally build those `<link>`s here, so we can keep configuration clean & without a need to transform user's input twice - there and here.

Reply:
Changed. I didn't want to introduce new class(es) to store data (`path` and `type`/`media`) and used `Pair` internally